index.html

```diff
@@ -214,17 +215,66 @@ <h3 class="center">Partners and Friends of FCCVienna</h3>
     <div class="container row">
       <h3 class="center">Contributors</h3>
       <p class="center">These are the people who have contributed actively more than once with content for our meetings. Without these creators, there would be no FCCVienna.</p>
-      <div class="col s12 m4 offset-m2">
-        <ul class="normalize-ul center">
-          <li><a href="https://matthiasliszt.github.io">Matthias Liszt</a></li>
-          <li><a href="https://github.com/42pre">Martin Plochberger</a></li>
-          <li><a href="https://github.com/edorivai">Edo Rivai</a></li>
-          <li><a href="https://github.com/bonny3d">Bonny Dongre</a></li>
-          <li><a href="https://github.com/abisz">Simon Reinsperger</a></li>
-          <li>Markus Hösel</li>
-        </ul>
+     
```

Comment:
can you keep the contributors list and just add one card as an example? Or grant me write access to your fork, so I can push my commits to your branch and merge it?